src/containers/MyNdla/Arena/components/ArenaForm.tsx

```diff
@@ -171,6 +171,13 @@ const ArenaForm = ({
         name="content"
         rules={{
           required: validationT({ type: 'required', field: 'content' }),
+          maxLength: {
+            value: 32767,
+            message: validationT({
+              type: 'maxLength',
+              field: 'content',
```

Comment:
Kunne kanskje hatt verdien som en konst.
```suggestion
            message: validationT({
              type: 'maxLength',
              field: 'content',
              vars: { count: 32767 },
```